src/exp_baseexp.jl

```diff
@@ -1,3 +1,10 @@
+"""
+    ExpMethodHigham2005Base()
+
+The same as `ExpMethodHigham2005` but follows `Base.exp` closer.
+
+"""
+
 struct ExpMethodHigham2005Base
```

Comment:
```suggestion
"""
struct ExpMethodHigham2005Base
```

No space is required to attach the docstring to the struct.